SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -283,7 +323,7 @@ struct Omega {
 struct Bsl {
   template <class MatrixType>
   static void get(const su2double* vorticity, unsigned short, const MatrixType&, CSAVariables& var) {
-    var.Omega = GeometryToolbox::Norm(3, vorticity);
+    var.Omega = GeometryToolbox::Norm(3, vorticity)*(1.0 + var.beta_PGO);
```

Comment:
I am not familiar with the PGOMGA model but my guess is that we can better fit it into the current modular structure than it is done here. E.g. under the `struct Omega {` you could create you own `struct PGOMGA {` and put down you omega contribution there. 

Currently also in the ComputeResidual routine there is specific PGOMGA contributions which would be great to move into specialized PGOMGA contributions following the current structure